File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = new int[2];
        int cnt1 = 0; //짝수의 개수
        int cnt2 = 0; //홀수의 개수
        
        for(int i = 0; i < num_list.length; i++) {
            if( num_list[i] % 2 == 0) {
                cnt1++;
            }
            if( num_list[i] % 2 == 1) {
                cnt2++;
            }
        }
        
        answer[0] = cnt1;
        answer[1] = cnt2;
        
        
        return answer;
    }
}

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    var answer = [];
    var num1 = 0;
    var num2 = 0;
    for(let i = 0; i < num_list.length; i++) {
        if(num_list[i] % 2 == 0) {
            num1++;
        } else {
            num2++;
        }
    }
    answer.push(num1);
    answer.push(num2);
    return answer;
}